### ISSUE TITLE
multiple backends > display improvements

### ISSUE THREAD

**Issue by dennisvandehoef** [camptocamp/terraboard#224]
I really love the feature that you now can have multiple backends (s3 buckets in our case).

Though while testing this I noticed that in the GUI there is no way to distinct which resource is defined in which backend.
This especially becomes a problem if the paths are looking alike.

Am I overseeing a configuration option to enable this in the GUI?

**Comment by raphink** [comment]
Indeed, I don't think there is a way to distinguish the sources in the GUI. It'd definitely be an interesting feature, for which we'd welcome a PR.

**Comment by Jaff** [comment]
We could definitely make use of this feature, as our org has more than 6 distinct accounts sharing primary bucket path. If anyone has thoughts on making a switchable front page based on account selection triggered by SSO login, I'd be willing to contribute.

**Comment by RafiShaik12345** [comment]
I have achieved multiple backends from the same provider but different accounts, like staging backend and development backend. Successfully able to pull the data from these backends on the tarraboard UI. but Im looking forward to achieve for differentiation those backends on the UI part. feel free to help me out on this blocker. thanks!

**Comment by kneemaa** [comment]
I too could use this, I've even gone as far as to query the DB directly and haven't found how we track the backend of each state
